File: SDK/KAICONG/DEMO POS无页模式/DEMO_C#/DEMO/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using POSDLL_V2;
using POSDLL;

namespace DEMO
{
    public partial class POSDLL_V1 : Form
    {
        public static POSDLL_V1 fmPOSDLL_V1; 
        public static int hPort = -1;
        string lpName = string.Empty;
        int nParam = -1;
        int nBaudrate = -1;
        int nDataBits = -1;
        int nStopBits = -1;
        int nParity = -1;
        string[] RAMimagePath = new string[2];

        public POSDLL_V1()
        {
            InitializeComponent();
            fmPOSDLL_V1 = this;
        }

        private void rbCom_CheckedChanged(object sender, EventArgs e)
        {
            cbDriveName.Enabled = false;
            cbComName.Enabled = true;
            cbDataBits.Enabled = true;
            cbParity.Enabled = true;
            cbStopBits.Enabled = true;
            cbStreamCtr.Enabled = true;
            cbBaudrate.Enabled = true;

        }

        private void rbDriveProgram_CheckedChanged(object sender, EventArgs e)
        {
            cbComName.Enabled = false;
            cbDataBits.Enabled = false;
            cbParity.Enabled = false;
            cbStopBits.Enabled = false;
            cbStreamCtr.Enabled = false;
            cbBaudrate.Enabled = false;
            cbDriveName.Enabled = true;
            nParam = C_POSDLL.POS_OPEN_PRINTNAME;
        }

        private void btOpen_Click(object sender, EventArgs e)
        {
            btOpen.Enabled = false;
            btStatus.Enabled = true;
            btPrint.Enabled = true;
            btClose.Enabled = true;
            hPort = C_POSDLL.POS_Open(lpName, nBaudrate, nDataBits, nStopBits, nParity, nParam);
            MessageBox.Show("0x" + hPort.ToString("x"));

        }

        private void btClose_Click(object sender, EventArgs e)
        {
            int rValue = C_POSDLL.POS_Close();
            MessageBox.Show("0x" + rValue.ToString("x"));
            btOpen.Enabled = true;
            btStatus.Enabled = false;
            btPrint.Enabled = false;
            btClose.Enabled = false;
        }

        private void cbDriveName_SelectedIndexChanged(object sender, EventArgs e)
        {
            lpName = cbDriveName.Text;
        }

        private void cbComName_SelectedIndexChanged(object sender, EventArgs e)
        {
            lpName = cbComName.Text;
        }

        private void cbStreamCtr_SelectedIndexChanged(object sender, EventArgs e)
        {
            nParam = cbStreamCtr.SelectedIndex;
        }

        private void btPrint_Click(object sender, EventArgs e)
        {
            
            C_POSDLL.POS_SetMode(0);
            C_POSDLL.POS_S_TextOut("POS Thermal Printer", 48, 0, 2, 0, 0x00);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("热敏打印机", 24, 0, 0, 0, 0x100);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_SetRightSpacing(12);
            C_POSDLL.POS_S_TextOut("热敏打印机", 24, 0, 0, 0, 0x80);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_SetRightSpacing(0);
            C_POSDLL.POS_S_TextOut("BM9000", 0, 0, 0, 1, 0x80);
            C_POSDLL.POS_S_TextOut("POS PRINTER", 192, 0, 0, 0, 0x00);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_SetRightSpacing(3);
            C_POSDLL.POS_S_TextOut("BM9000", 0, 0, 0, 1, 0x80);
            C_POSDLL.POS_S_TextOut("POS PRINTER", 192, 0, 0, 0, 0x00);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_SetRightSpacing(6);
            C_POSDLL.POS_S_TextOut("BM9000", 0, 0, 0, 1, 0x80);
            C_POSDLL.POS_S_TextOut("POS PRINTER", 192, 0, 0, 0, 0x00);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("正常字体打印", 0, 0, 0, 0, 0x00);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("反显字体打印", 0, 0, 0, 0, 0x400);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("顺时针旋转90度字体打印", 0, 0, 0, 0, 0x1000);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("倒置字体打印", 0, 0, 0, 0, 0x200);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("    ----------------> Logo 1", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_PreDownloadBmpToRAM(RAMimagePath[0], 0);
            C_POSDLL.POS_S_PrintBmpInRAM(0, 96, 0);
            C_POSDLL.POS_S_TextOut("    ----------------> Logo 2", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_PreDownloadBmpToRAM(RAMimagePath[1], 0);
            C_POSDLL.POS_S_PrintBmpInRAM(0, 0, 3);
            C_POSDLL.POS_S_TextOut("    ----------------> Logo 3", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_PrintBmpInFlash(1, 96, 0);
            C_POSDLL.POS_S_TextOut("UPC-A", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("01234567890", 24, 0x41, 3, 100, 0x00, 0x02, 11);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("UPC-E", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("042100005264", 96, 0x42, 3, 100, 0x00, 0x02, 12);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("EAN13", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("104210000526", 24, 0x43, 3, 100, 0x00, 0x02, 12);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("EAN8", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("2042100", 24, 0x44, 3, 100, 0x00, 0x02, 7);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("CODE39", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("0423", 0, 0x45, 2, 100, 0x00, 0x02, 4);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("ITF", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("0423", 96, 0x46, 3, 100, 0x00, 0x02, 4);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("CODEBAR", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("A42368B", 24, 0x47, 3, 100, 0x00, 0x02, 7);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("CODE93", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("342368ABC", 24, 0x48, 3, 100, 0x00, 0x02, 9);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_TextOut("CODE128", 0, 0, 0, 0, 0);
            C_POSDLL.POS_FeedLine();
            C_POSDLL.POS_S_SetBarcode("{AHI{C345678", 0, 0x49, 2, 100, 0x00, 0x02, 12);
            C_POSDLL.POS_FeedLine();

            C_POSDLL.POS_SetMotionUnit(180, 180);
            C_POSDLL.POS_PL_SetArea(0, 0, 384, 740, 0);
            C_POSDLL.POS_SetMode(1);
            C_POSDLL.POS_PL_TextOut("PageMode:Thermal Printer", 0, 32, 0, 0, 0, 0);
            C_POSDLL.POS_PL_TextOut("页模式：热敏打印机测试页", 0, 96, 0, 0, 0, 0);
            C_POSDLL.POS_PL_TextOut("反显", 96, 128, 0, 0, 0, 0x400);
            C_POSDLL.POS_PL_TextOut("下划线", 96, 160, 0, 0, 0, 0x100);
            C_POSDLL.POS_PL_TextOut("    ----------------> Logo 4", 0, 192, 0, 0, 0, 0);
            C_POSDLL.POS_PreDownloadBmpToRAM(RAMimagePath[0], 0);
            C_POSDLL.POS_PL_PrintBmpInRAM(0, 48, 465, 0);
            C_POSDLL.POS_PL_SetBarcode("{AHI{C345678", 0, 627, 0x49, 2, 162, 0x00, 0x02, 12);
            C_POSDLL.POS_PL_TextOut("页模式：热敏打印机测试页 end", 0, 699, 0, 0, 0, 0);
            C_POSDLL.POS_PL_Print();
            C_POSDLL.POS_PL_Clear();
            
        }

        private void cbParity_SelectedIndexChanged(object sender, EventArgs e)
        {
            nParity = cbParity.SelectedIndex;
        }

        private void cbStopBits_SelectedIndexChanged(object sender, EventArgs e)
        {
            nStopBits = cbStopBits.SelectedIndex;
        }

        private void cbBaudrate_SelectedIndexChanged(object sender, EventArgs e)
        {
            nBaudrate = int.Parse(cbBaudrate.Text);
        }

        private void cbDataBits_SelectedIndexChanged(object sender, EventArgs e)
        {
            nDataBits = int.Parse(cbDataBits.Text);
        }

        private void POSDLL_V1_Load(object sender, EventArgs e)
        {
            cbComName.Items.AddRange(System.IO.Ports.SerialPort.GetPortNames());
            foreach (string i in System.Drawing.Printing.PrinterSettings.InstalledPrinters)
                cbDriveName.Items.Add(i);
        }

        private void cbPageWidth_SelectedIndexChanged(object sender, EventArgs e)
        {
            if (cbPageWidth.Text == "56mm")
            {
                RAMimagePath[0] = "Look.bmp";
                RAMimagePath[1] = "Kitty.bmp";
            }
            else if (cbPageWidth.Text == "80mm")
            {
                RAMimagePath[0] = "刀剑神域单色位图.bmp";
                RAMimagePath[1] = "羊.jpg";
            }
            else
                MessageBox.Show("请选择页宽");

        }

        private void btStatus_Click(object sender, EventArgs e)
        {
            MessageBox.Show(C_POSDLL.POS_RTQueryStatus());
        }

        private void btShotPrint_Click(object sender, EventArgs e)
        {
            this.Hide();
            Form2 form2 = new Form2();
            form2.InstanceRef = this;
            form2.Show();
        }

        

    }
}

File: SDK/KAICONG/DEMO POS有页模式/DEMO_VB/DEMO_VB/Form1.vb
Public Class Form1
    Public lpName As String
    Public nParam As Int32
    Public nBaudrate As Int32
    Public nDataBits As Int32
    Public nStopBits As Int32
    Public nParity As Int32
    Public RAMimagePatha As String
    Public RAMimagePathb As String

    Private Sub btOpen_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btOpen.Click
        btOpen.Enabled = False
        btPrint.Enabled = True
        btClose.Enabled = True
        Dim rValue As Int32
        rValue = POSDLL_V1.C_POSDLL.POS_Open(lpName, nBaudrate, nDataBits, nStopBits, nParity, nParam)
        MessageBox.Show("0x" + rValue.ToString("x"))

    End Sub

    Private Sub rbCom_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles rbCom.CheckedChanged
        cbDriveName.Enabled = False
        cbComName.Enabled = True
        cbDataBits.Enabled = True
        cbParity.Enabled = True
        cbStopBits.Enabled = True
        cbStreamCtl.Enabled = True
        cbBaudrate.Enabled = True
    End Sub

    Private Sub rbDriveName_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles rbDriveName.CheckedChanged
        cbComName.Enabled = False
        cbDataBits.Enabled = False
        cbParity.Enabled = False
        cbStopBits.Enabled = False
        cbStreamCtl.Enabled = False
        cbBaudrate.Enabled = False
        cbDriveName.Enabled = True
        nParam = POSDLL_V1.C_POSDLL.POS_OPEN_PRINTNAME
    End Sub

    Private Sub btClose_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btClose.Click
        Dim rValue As Int32
        rValue = POSDLL_V1.C_POSDLL.POS_Close()
        MessageBox.Show("0x" + rValue.ToString("x"))
        btOpen.Enabled = True
        btPrint.Enabled = False
        btClose.Enabled = False
    End Sub

    Private Sub cbComName_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbComName.SelectedIndexChanged
        lpName = cbComName.Text
    End Sub

    Private Sub cbDriveName_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbDriveName.SelectedIndexChanged
        lpName = cbDriveName.Text
    End Sub

    Private Sub cbStreamCtl_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbStreamCtl.SelectedIndexChanged
        nParam = cbStreamCtl.SelectedIndex
    End Sub

    Private Sub btPrint_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btPrint.Click
        POSDLL_V1.C_POSDLL.POS_SetMode(0)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("KaiCong POS Thermal Printer", 48, 0, 2, 0, &H0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("开聪热敏打印机", 24, 0, 0, 0, &H100)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_SetRightSpacing(12)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("开聪热敏打印机", 24, 0, 0, 0, &H80)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_SetRightSpacing(0)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("BM9000", 0, 0, 0, 1, &H80)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("POS PRINTER", 192, 0, 0, 0, &H0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_SetRightSpacing(3)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("BM9000", 0, 0, 0, 1, &H80)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("POS PRINTER", 192, 0, 0, 0, &H0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_SetRightSpacing(6)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("BM9000", 0, 0, 0, 1, &H80)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("POS PRINTER", 192, 0, 0, 0, &H0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("正常字体打印", 0, 0, 0, 0, &H0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("反显字体打印", 0, 0, 0, 0, &H400)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("顺时针旋转90度字体打印", 0, 0, 0, 0, &H1000)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("倒置字体打印", 0, 0, 0, 0, &H200)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("    ----------------> Logo 1", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_PreDownloadBmpToRAM(RAMimagePatha, 0)
        POSDLL_V1.C_POSDLL.POS_S_PrintBmpInRAM(0, 96, 0)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("    ----------------> Logo 2", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_PreDownloadBmpToRAM(RAMimagePathb, 0)
        POSDLL_V1.C_POSDLL.POS_S_PrintBmpInRAM(0, 0, 3)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("    ----------------> Logo 3", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_PrintBmpInFlash(1, 96, 0)
        POSDLL_V1.C_POSDLL.POS_S_TextOut("UPC-A", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("01234567890", 24, &H41, 3, 100, &H0, &H2, 11)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("UPC-E", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("042100005264", 96, &H42, 3, 100, &H0, &H2, 12)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("EAN13", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("104210000526", 24, &H43, 3, 100, &H0, &H2, 12)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("EAN8", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("2042100", 24, &H44, 3, 100, &H0, &H2, 7)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("CODE39", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("*0423*", 0, &H45, 2, 100, &H0, &H2, 6)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("ITF", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("0423", 96, &H46, 3, 100, &H0, &H2, 4)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("CODEBAR", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("A42368B", 24, &H47, 3, 100, &H0, &H2, 7)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("CODE93", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("342368ABC", 24, &H48, 3, 100, &H0, &H2, 9)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_TextOut("CODE128", 0, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_FeedLine()
        POSDLL_V1.C_POSDLL.POS_S_SetBarcode("{AHI{C345678", 0, &H49, 2, 100, &H0, &H2, 12)
        POSDLL_V1.C_POSDLL.POS_FeedLine()

        POSDLL_V1.C_POSDLL.POS_SetMotionUnit(180, 180)
        POSDLL_V1.C_POSDLL.POS_PL_SetArea(0, 0, 384, 740, 0)
        POSDLL_V1.C_POSDLL.POS_SetMode(1)
        POSDLL_V1.C_POSDLL.POS_PL_TextOut("PageMode:KaiCongDianZi Thermal Printer", 0, 32, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_PL_TextOut("页模式：开聪电子热敏打印机测试页", 0, 96, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_PL_TextOut("反显", 96, 128, 0, 0, 0, &H400)
        POSDLL_V1.C_POSDLL.POS_PL_TextOut("下划线", 96, 160, 0, 0, 0, &H100)
        POSDLL_V1.C_POSDLL.POS_PL_TextOut("    ----------------> Logo 4", 0, 192, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_PreDownloadBmpToRAM(RAMimagePatha, 0)
        POSDLL_V1.C_POSDLL.POS_PL_PrintBmpInRAM(0, 48, 465, 0)
        POSDLL_V1.C_POSDLL.POS_PL_SetBarcode("{AHI{C345678", 0, 627, &H49, 2, 162, &H0, &H2, 12)
        POSDLL_V1.C_POSDLL.POS_PL_TextOut("页模式：开聪电子热敏打印机测试页 end", 0, 699, 0, 0, 0, 0)
        POSDLL_V1.C_POSDLL.POS_PL_Print()
        POSDLL_V1.C_POSDLL.POS_PL_Clear()
    End Sub

    Private Sub cbParity_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbParity.SelectedIndexChanged
        nParity = cbParity.SelectedIndex
    End Sub

    Private Sub cbStopBits_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbStopBits.SelectedIndexChanged
        nStopBits = cbStopBits.SelectedIndex
    End Sub

    Private Sub cbBaudrate_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbBaudrate.SelectedIndexChanged
        nBaudrate = Int32.Parse(cbBaudrate.Text)
    End Sub

    Private Sub cbDataBits_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cbDataBits.SelectedIndexChanged
        nDataBits = Int32.Parse(cbDataBits.Text)
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        cbComName.Items.AddRange(System.IO.Ports.SerialPort.GetPortNames())
        RAMimagePatha = "Kitty.bmp"
        RAMimagePathb = "Look.bmp"
    End Sub
End Class
